composites/HelpCenter/views/VideoTutorial.js

```diff
@@ -100,22 +104,28 @@ VideoDescriptionItem.defaultProps = {
  * @returns {ReactElement} The VideoTutorial component.
  */
 export default function VideoTutorial( props ) {
+	const hasParagraphs = props.paragraphs.length > 0;
+
 	return (
 		<VideoTutorialContainer className={ `${ props.className }__container` }>
-			<VideoContainer className={ `${ props.className }__video-container` }>
+			<VideoContainer
+				className={ `${ props.className }__video-container` }
+				hasParagraphs={ hasParagraphs }
+			>
 				<YouTubeVideo
 					src={ props.src }
-					title={ props.title } />
+					title={ props.title }
+				/>
 			</VideoContainer>
-			<VideoDescriptions className={ `${ props.className }__descriptions` }>
+			{ hasParagraphs && <VideoDescriptions className={ `${ props.className }__descriptions` }>
```

Comment:
This is hard to read, when hasPararaph is true, the component will be rendered I guess? It might be a better idea to wrap this in a function: `renderVideoDescriptions`. In that function you can do an early `return null` and returns the rendered videodescriptions as default like:

```js
function renderVideoDescriptions( props ) {
     if(  props.paragraphs.length > 0 ) {
        return null;
     }

     // Render the component.
     ....
} 
```

Reply:
Hm I tend to disagree. This is pretty standard in Gutenberg, and personally I think it's an elegant and concise way to render conditionally. Why we should return null?